### ISSUE TITLE
[Accessibility]: Megamenu dropdown brackground doesn't scale with browser font size settings causing text contrast issues

### ISSUE THREAD

**Issue by qjac** [department-of-veterans-affairs/vets-design-system-documentation#5498]
### Point of contact

Jacqueline Quintanilla

### How severe do you think this issue is?

Critical

### Severity rationale

Users who have browser font setting set to enlarge may not be able to read the text when it overflows the background due to contrast issues.

### URL where issue occurs

https://www.va.gov/

### Component or pattern (if applicable)

Header

### Environment details

OS: macOS
Browser: Chrome (with font setting set to extra large)

### Steps to reproduce

1. Using Chrome, go to the Settings, search for “font,” and increase the font size to “Very Large.” 
2. Go to [homepage](https://www.va.gov/) or any page at desktop screen size
3. Click a top-level main nav item to open the dropdown
4. See that the enlarged menu text overflows the background.

### Observed behavior

Enlarged menu text in dropdown overflows the background container.

### Expected behavior / Acceptance criteria

Background container scales or text wraps to stay within container to avoid text overlaying other text or background colors that cause contrast issues.

### Supporting evidence (screenshots, recordings, code snippets)

<img width="703" height="842" alt="Image" src="https://github.com/user-attachments/assets/ba547281-93cd-4a30-8dca-79c2cab7a9f3" />

### Related issues or systemic notes

[Known issue discussed in slack thread 🔒](https://dsva.slack.com/archives/C0335Q88X0S/p1757097106658609)

### Type of issue

- [ ] Axe-core
- [ ] Cognitive | user flows & navigating
- [x] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Content style guide
- [ ] Focus
- [ ] Headings
- [ ] Keyboard
- [ ] Link and button labels
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Screenreader | assistive tech & device support
- [ ] Semantic HTML
- [ ] URL standards
- [x] Zoom or Magnification
- [ ] Something else
